metagraph/src/graph/annotated_graph_algorithm.cpp

```diff
@@ -90,32 +92,44 @@ mask_nodes_by_unitig_labels(const AnnotatedDBG &anno_graph,
 
         return (in_count >= label_in_factor * path.size())
             && (other_count <= label_other_fraction * (in_count + out_count + other_count));
-    });
+    }, num_threads);
 }
 
 sdsl::int_vector<> fill_count_vector(const AnnotatedDBG &anno_graph,
```

Comment:
maybe `fill_diff_label_count_vector`?

Reply:
there's no diff happening at this point, so how about just `construct_label_count_vector`?

Reply:
But there are in and out groups, so I thought about DIFFerential counts

Reply:
how about `construct_interleaved_label_count_vector`?

Reply:
Hm, try something else?

Reply:
or `construct_in_out_label_count_vector`? If not, we can do `construct_diff_label_count_vector`. This is not in the external API anyway, so it's name is not terribly important.